app/assets/javascripts/_page_show.es6

```diff
@@ -1,3 +1,9 @@
+const COMPONENT_CLASSES = [
```

Comment:
nice refactor! This looks neat. 

For removing the images from external link styling, you could do something like `.page-compound-body-component p` to get the WYSIWIG paragraphs. For the featured link, you could throw a class on the `a` tag and then add something like `.page-compound-body-component .featured-url` to this list

Reply:
> nice refactor! This looks neat.
> 
> For removing the images from external link styling, you could do something like `.page-compound-body-component p` to get the WYSIWIG paragraphs. For the featured link, you could throw a class on the `a` tag and then add something like `.page-compound-body-component .featured-url` to this list

This issue has been resolved! Nice catch 😄